core/src/test/java/apoc/load/LoadJsonTest.java

```diff
@@ -414,4 +421,10 @@ public void testLoadJsonParams() throws Exception {
                     assertFalse("value should be not empty", value.isEmpty());
                 });
     }
+
+    @Test
+    public void shouldTerminateLoadWhenTransactionIsTimedOut() {
+        final String query = "CALL apoc.load.json('https://github.com/knowitall/yelp-dataset-challenge/blob/master/data/yelp_phoenix_academic_dataset/yelp_academic_dataset_review.json?raw=truehttps://github.com/knowitall/yelp-dataset-challenge/blob/master/data/yelp_phoenix_academic_dataset/yelp_academic_dataset_review.json?raw=true')";
```

Comment:
I'd prefer you didn't reference external resources in case it is deleted in the future, is it possible to add to this repo instead?